setuptools/lib2to3_ex.py

```diff
@@ -33,6 +34,12 @@ def run_2to3(self, files, doctests=False):
             return
         if not files:
             return
+
+        warnings.warn(
+            "2to3 support is deprecated. Please migrate to "
+            "a single-codebase solution or roll your own "
+            "conversion process.",
```

Comment:
Possibly "if you still need to support a python 2"?

Reply:
Done.